source/test/F0.Minesweeper.Components.Tests/GameSnapshotTest.razor

```diff
@@ -0,0 +1,30 @@
+@inherits TestComponentBase
+
+<SnapshotTest Description="Verifies the correct markup of the Game component."
+			  Timeout="TimeSpan.FromSeconds(2)"
+			  Setup="Setup">
```

Comment:
Interesting piece for snapshot tests: they provide a setup but no teardown. Tbh for me that's still acceptable since I personally want to use it only for nicer markup tests and not for functional unit tests. What are your thoughts?

Reply:
I think there was a remark somewhere to use "dispose" instead of "tear-down". 
I'm not sure

Reply:
Yes, I know. But that's only valid for tests implementing `TestContext`. Snapshot Tests implement the `TestComponentBase` which to my knowledge has nothing like this